build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -19,26 +46,39 @@ export abstract class PackageCommand<
 	T extends typeof Command & { flags: typeof PackageCommand.flags },
 > extends BaseCommand<T> {
 	static flags = {
+		all: Flags.boolean({
+			char: "a",
+			description:
+				"Run on all packages and release groups. Cannot be used with --dir, --packages, or --releaseGroup.",
+			exclusive: ["dir", "packages", "releaseGroup"],
+		}),
 		dir: Flags.directory({
 			char: "d",
 			description:
-				"Run on the package in this directory. Cannot be used with --releaseGroup or --packages.",
-			exclusive: ["packages", "releaseGroup"],
+				"Run on the package in this directory. Cannot be used with --all, --packages, or --releaseGroup.",
+			exclusive: ["packages", "releaseGroup", "all"],
 		}),
 		packages: Flags.boolean({
```

Comment:
We should probably leave it for a separate PR if we change it, but I find "packages" a confusing option. Renaming this to "independent" or "separate" or "non-grouped" seems like it would be better.

Reply:
Good point. I'll do that in a separate PR.